src/Microsoft.AspNetCore.Mvc.TagHelpers/Cache/DistributedCacheTagHelperFormatter.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace Microsoft.AspNetCore.Mvc.TagHelpers.Cache
+{
+    /// <summary>
+    /// Implements <see cref="IDistributedCacheTagHelperFormatter"/> by serializing the content
+    /// in UTF8.
+    /// </summary>
+    public class DistributedCacheTagHelperFormatter : IDistributedCacheTagHelperFormatter
+    {
+        
+        /// <inheritdoc />
+        public Task<byte[]> SerializeAsync(DistributedCacheTagHelperFormattingContext context)
+        {
+            if (context == null) 
+            {
+                throw new ArgumentNullException(nameof(context));    
+            }
+            
+            if (context.HtmlString == null) 
+            {
+                throw new ArgumentNullException(nameof(context.HtmlString));    
+            }
+            
+            var serialized = Encoding.UTF8.GetBytes(context.HtmlString.ToString());
```

Comment:
Hmm, should we be using `WriteTo` here instead of `ToString()`? `HtmlString` is an `IHtmlCOntent` after all